binance_trade_bot/config.py

```diff
@@ -173,3 +157,6 @@ def __init__(self):
 
         allow_coin_merge = os.environ.get("ALLOW_COIN_MERGE") or config.get(USER_CFG_SECTION, "allow_coin_merge")
         self.ALLOW_COIN_MERGE = str(allow_coin_merge).lower() == 'true'
+
+        self.USE_MARGIN = os.environ.get("USE_MARGIN") or config.get(USER_CFG_SECTION, "use_margin")
```

Comment:
We already have some bool flags in the config values which use "true" and "false". please change it to "true"/"false" notation. You could copy the code from the "allow_coin_merge" setting.